app/Actions/Review/CreateReviewAction.php

```diff
@@ -46,6 +49,8 @@ public function execute(CreateReviewRequest $request): CreateReviewResponse
         );
 
         broadcast(new ReviewAddEvent($review))->toOthers();
+        $this->userRepository->getById($place->creator_id)
```

Comment:
better notation is
$user = $this->userRepository->getById($place->creator_id);
$user->notify(new ReviewPlaceNotification($place, Auth::user()));